spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/data/neo4j/Neo4jDataAutoConfigurationTests.java

```diff
@@ -94,6 +94,7 @@ public void customSessionFactory() {
 		this.contextRunner.withUserConfiguration(CustomSessionFactory.class).run((context) -> {
 			assertThat(context).doesNotHaveBean(org.neo4j.ogm.config.Configuration.class);
 			assertThat(context).hasSingleBean(SessionFactory.class);
+			assertThat(context).hasSingleBean(OpenSessionInViewInterceptor.class);
```

Comment:
Did you intend to commit this?

Reply:
Yes. I first had a dedicated test `customSessionShouldNotDisableOSiV`, my interpretation of 

> (i.e. a test that provides a custom SessionFactory and validates the necessary bits are still auto-configured.